web/components/feed/feed-answer-comment-group.tsx

```diff
@@ -54,13 +53,22 @@ export function FeedAnswerCommentGroup(props: {
       answerComments.map((c) => c.id).includes(comment.replyToCommentId)
   )
   const commentsList = answerComments.concat(commentReplies)
-
-  const prob = getDpmOutcomeProbability(contract.totalShares, answer.id)
-  const probPercent = formatPercent(prob)
+  const thisAnswerProbOnMount = useRef<number>()
+  const thisAnswerProb = bets
+    .filter((bet) => bet.outcome === answer.number.toString())
+    .sort((a, b) => b.createdTime - a.createdTime)[0].probAfter
+  const probPercent = formatPercent(thisAnswerProb)
   const betsByCurrentUser = (user && betsByUserId[user.id]) ?? []
   const commentsByCurrentUser = (user && commentsByUserId[user.id]) ?? []
   const isFreeResponseContractPage = !!commentsByCurrentUser
+
+  useEffect(() => {
+    thisAnswerProbOnMount.current = thisAnswerProb.valueOf()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
   useEffect(() => {
+    if (thisAnswerProbOnMount.current === thisAnswerProb) return
```

Comment:
It seems like the logic is you only want to scroll if the most recent bet is made after you loaded the page?

Couldn't you save the time at first render, and then if `mostRecentCommentableBet.createdTime` is greater, proceed with scrolling?

Seems like you would calculate the `mostRecentCommentableBet` outside the effect, and then add have deps of `[mostRecentCommentableBet, showReply, answer, scrollAndOpenReplyInput]`.

And you'd wrap `scrollAndOpenReplyInput` in a useEvent so the ref doesn't change.

Reply:
Great! that works and I suppose is better than prob as in the far future when there is as much liquidity as the oceans on earth, the probability might not change with one measly bet...

Reply:
Finally got useEvent to work
https://github.com/reactjs/rfcs/blob/useevent/text/0000-useevent.md